vector/src/main/codegen/templates/ComplexCopier.java

```diff
@@ -41,15 +41,8 @@ public class ComplexCopier {
    * @param input field to read from
    * @param output field to write to
    */
-  public static void copy(FieldReader input, FieldWriter output) {
-    writeValue(input, output, null);
-  }
-
-  public static void copy(FieldReader input, FieldWriter output, ExtensionTypeWriterFactory extensionTypeWriterFactory) {
```

Comment:
Removing this method would be a backward incompatible change, it may be better to actually deprecate it and documenting the pattern, to be removed at a later time (see `ComplexWriters` for example)